server/tests/test_notebook_revisions_cleanup.py

```diff
@@ -43,6 +49,25 @@ def notebook_and_draft_revisions(fake_user):
     return notebook, notebook_revisions
 
 
+@pytest.fixture
+def notebook_and_duplicate_draft_revisions(fake_user):
```

Comment:
I would just call this `notebook_with_duplicate_draft_revisions` and then only return the notebook object from this fixture. It looks like you don't actually use the notebook_revisions list in your test.

Reply:
Sure